src/app/detail/ui/ActivityLog.tsx

```diff
@@ -0,0 +1,35 @@
+import ActivityDescription from '@/components/activity/ActivityDescription'
+import { ActivityType } from '@/types/interfaces'
+
+import { Avatar, Box, Stack } from '@mui/material'
+import { BoldTypography, StyledTypography, VerticalLine, WrapperStack } from '@/app/detail/ui/styledComponent'
+import { getTimeDifference } from '@/utils/getTimeDifference'
+
+interface Prop {
+  log: any
```

Comment:
We can do better than any... do we not have an interface for log?

Reply:
@foleyatwork I have created schema and types for the logs in [this](https://github.com/copilot-platforms/tasks-app/pull/68) PR. After it gets merged, @arpandhakal will be utilizing the types to get rid of all `any`.